bin/cli.js

```diff
@@ -222,13 +249,16 @@ function packageApp() {
   });
 }
 
+function displayVersion() {
+  console.log(packageJson.version);
```

Comment:
It'd be nice if `--version` also displayed the version of the runtime, but we can do this in a followup.